src/components/NavBar.vue

```diff
@@ -114,6 +118,25 @@ export default defineComponent({
       menuOpen: false,
     };
   },
+  watch: {
+    startScheduleGeneratorTour() {
+      const scheduleGeneratorTour = introJs();
+      scheduleGeneratorTour.setOptions({
+        steps: [
+          {
+            element: '#schedule-generator',
+            intro: `<div class="introjs-tooltipTop"><div class="introjs-customTitle">Introducing a New Page</div></div>
+          <div class = "introjs-bodytext">Use Build 💪 to automatically generate new schedules based on the courses your want to take! 📆</div>`,
```

Comment:
I think there's a typo here - should be "generate new schedules based on the courses **you** want to take!" Other than that great job!!